app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def increment_counter
+    current_post = Post.find(post_id)
+    current_post.update(comments_counter: (current_post.comments_counter || 0) + 1)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] This approach works well, however it would be nice to use `after_save` Rails callback to call the `increment_counter` method to increase your comments_counter each time a new comment is created 👍 
### Hint:
**In order to do this, you'd have to make your `increment_counter` method private**
So you'd have something like this 👇 
```
after_save :increment_counter

private

def increment_counter
  current_post = Post.find(post_id)
  current_post.update(comments_counter: (current_post.comments_counter || 0) + 1)
end
```